index.js

```diff
@@ -362,6 +364,14 @@
         }
       });
 
+  //  keys :: StrMap a -> Array String
+  var keys = function(strMap) {
+    if (typeof Object.keys === 'function') return Object.keys(strMap);
```

Comment:
I would advice against optimizing performance before

1. It's established that there is a performance issue
2. It's established that the optimization fixes it
3. The optimized code [behaves exactly the same](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys#Polyfill) as the non-optimized code

That said, a quick performance benchmark on my local machine shows that `Object.keys` is about *5 times as fast* as `for ... in hasOwn`, so that certainly crosses out `2` from my list. ;)

I'm a little worried about the behavior differences between `Object.keys` and `for ... in` as illustrated by the MDN suggested polyfill. Most of it seems to be related to some bug where `for ... in` misses some properties that *should* have been enumerated though. It probably won't affect us.

My vote goes in favor of the `Object.keys` branch.